client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -138,10 +142,18 @@ export default class ProductFormDetailsCard extends Component {
 							<FormLabel htmlFor="description">{ __( 'Description' ) }</FormLabel>
 							{ this.renderTinyMCE() }
 						</FormFieldSet>
+						{ isNumber( product.id ) && config.isEnabled( 'woocommerce/extension-reviews' ) &&
+						<ProductReviewsWidget site={ site } product={ product } /> }
 					</div>
 				</div>
 			</Card>
 		);
 	}
-
 }
+
+export default connect( ( state ) => {
+	const site = getSelectedSiteWithFallback( state );
```

Comment:
At the GM we also talked about when to connect a component and when to pass the site object as a prop. IIRC we didn't really have a good answer, but my feeling here is that the new `ProductReviewsWidget` component should be the connected one, and only product would be passed in? Feel free to ignore this if you disagree :)